swagger/public_v1/component_schemas/SubjectAttributes.yml

```diff
@@ -1,12 +1,22 @@
 ---
 description: "This schema is used to describe a subject."
 type: object
-required:
-  - code
 properties:
   name:
     type: string
     description: "Subject name."
   code:
     type: string
     description: "Unique subject code."
+  bursary_amount:
+    type: string
+    description: "non-repayable sum of money that universities can award to students to incentivise and support their studies"
+  early_career_payments:
+    type: string
+    description: "Eligible mathematics, physics, chemistry and languages teachers can apply for early-career payments of up to £9,000 after tax."
```

Comment:
should we actually be stating the subjects
if it changes then this needs to change